index.html

```diff
@@ -106,8 +106,9 @@
         <div class="action">
           <a class="button state-link" href="#" target="_blank">Register to Vote</a>
         </div>
+         <div class="embed-responsive embed-responsive-16by9">
```

Comment:
This shouldn't be necessary yet. Looks like this was copy and pasted from somewhere else. Also, when you use a `<div>` tag be sure to close it using `</div>`.